src/main/java/net/snowflake/ingest/streaming/internal/InternalParameterProvider.java

```diff
@@ -31,4 +31,9 @@ boolean setIcebergSpecificFieldsInEp() {
     // in the EP metadata, createdOn, and extendedMetadataSize.
     return isIcebergMode;
   }
+
+  boolean getComputeExtendedMetadataSize() {
```

Comment:
no need for a new method here, can reuse setIcebergSpecificFieldsInEp

Reply:
Removed.